### DISCUSSION TITLE
Missing docker images

### DISCUSSION THREAD

**Discussion by TrainmasterHD** [WeblateOrg/weblate#12838] [General] [answered]
Hi,
I am missing the docker images for the recently released Weblate Versions(5.8 and 5.8.1). As long as they are not released, we are not able to update our self hosted Weblate instance.

Does it have a reason why they are not released yet?

Thanks for help

**Comment by nijel** [comment, marked as answer]
The image for 5.8.1 is there since today (see https://github.com/WeblateOrg/docker/issues/2741), see https://github.com/WeblateOrg/docker/pkgs/container/weblate.

5.8 was not published on PyPI due an issue in Python package build, so the Docker image can't be built for it.
